rlbottraining/exercise_runner.py

```diff
@@ -90,7 +90,8 @@ def run_module(python_file_with_playlist: Path, history_dir: Optional[Path] = No
         for seed in infinite_seed_generator():
             playlist = playlist_factory()
             wrapped_exercises = [TrainingExerciseAdapter(ex) for ex in playlist]
-            result_iter = rlbot_run_exercises(setup_manager, wrapped_exercises, seed)
+            reload_agent = reload_policy != ReloadPolicy.NEVER
+            result_iter = rlbot_run_exercises(setup_manager, wrapped_exercises, seed, reload_agent)
```

Comment:
I'd recommend writing it as `reload_agent=reload_agent` just in case we add another keyword argument before this one.
Please update the requirements on RLBot version as the extra argument is required.